src/transformers/quantizers/quantizer_compressed_tensors.py

```diff
@@ -117,9 +117,19 @@ def _process_model_before_weight_loading(self, model, **kwargs):
         ct_quantization_config = self.compressor.quantization_config
 
         if self.run_compressed:
-            if not self.is_quantization_compressed:
-                raise ValueError("`run_compressed` is only supported for quantized_compressed models")
-            apply_quantization_config(model, ct_quantization_config, run_compressed=True)
+            if self.is_sparsification_compressed:
```

Comment:
We've tested this against our transformers/ci tests?

Reply:
All relevant tests this PR should fix are green; there are some failure(s) related to other fixes.